go.mod

```diff
@@ -0,0 +1,17 @@
+module go.viam.com/slam
+
+go 1.19
+
+require (
+	github.com/edaniels/golog v0.0.0-20230215213219-28954395e8d0
+	github.com/pkg/errors v0.9.1
+	go.viam.com/test v1.1.0
+)
+
+require (
+	github.com/benbjohnson/clock v1.1.0 // indirect
```

Comment:
Is there a reason why we are using a different version than RDK? https://github.com/viamrobotics/rdk/blob/main/go.mod#L141d